jreleaser.yml

```diff
@@ -17,10 +17,17 @@ deploy:
       strict: false
     mavenCentral:
       sonatype:
-        active: ALWAYS
+        active: RELEASE_PRERELEASE
```

Comment:
This should probably be just `RELEASE`.

Reply:
What is PRERELEASE for? I thought it would be used for Alphas, CRs, etc